src/policy/ephemeral_policy.h

```diff
@@ -50,8 +50,9 @@ bool PreCheckValidEphemeralTx(const CTransaction& tx, CFeeRate dust_relay_rate,
 /** Must be called for each transaction(package) if any dust is in the package.
  *  Checks that each transaction's parents have their dust spent by the child,
  *  where parents are either in the mempool or in the package itself.
- *  @returns std::nullopt if all dust is properly spent, or the txid of the violating child spend.
+ *  child_state and child_txid set on failure.
```

Comment:
f306a940d2ba470a13120853ace49509a8e496a6

looks odd, perhaps a pasting issue.

Reply:
It's a sentence fragment, I can touch it up to be grammatically correct

Reply:
touched up